AdyenComponents/BACS Direct Debit/Scenes/Input/BACSInputFormViewController.swift

```diff
@@ -0,0 +1,40 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import UIKit
+
+internal protocol BACSInputFormViewProtocol: FormViewProtocol {}
+
+internal class BACSInputFormViewController: FormViewController, BACSInputFormViewProtocol {
```

Comment:
this class is not needed, since the `presenter` can set itself as the `ViewControllerDelegate`, this `FormViewController` subclass adds no value.